kornia/feature/lightglue_onnx/lightglue.py

```diff
@@ -42,25 +43,35 @@ class OnnxLightGlue:
 
     required_data_keys: ClassVar[list[str]] = ["image0", "image1"]
 
-    def __init__(self, weights: str = "disk_fp16", device: Device = None) -> None:
+    def __init__(self, weights: str | None = None, device: Device = None) -> None:
         KORNIA_CHECK(ort is not None, "onnxruntime is not installed.")
         KORNIA_CHECK(np is not None, "numpy is not installed.")
 
         if device is None:
-            device = torch.device("cuda")
+            device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
why do not default to "cpu" which 100% everyone will have ?

Reply:
Good point!